creusot-contracts-dummy/src/lib.rs

```diff
@@ -35,7 +35,12 @@ pub fn snapshot(_: TS1) -> TS1 {
 
 #[proc_macro]
 pub fn ghost(_: TS1) -> TS1 {
-    quote::quote! { creusot_contracts::ghost::Ghost::from_fn(|| std::process::abort()) }.into()
+    quote::quote! { () }.into()
+}
+
+#[proc_macro]
+pub fn gh(_: TS1) -> TS1 {
+    quote::quote! { creusot_contracts::ghost::GhostBox::from_fn(|| std::process::abort())}.into()
 }
```

Comment:
If `gh` is just an alias for ghost I think you can define it only once in the leaf crate by doing `pub use ghost as gh`.

Reply:
It does a bit more, right now it's syntax sugar: `gh!(e) := ghost!(GhostBox::new(e))`

Reply:
We talked a bit with @jhjourdan, and he convinced me that:
- There should only be one macro (I tend to prefer `ghost!`)
- This macro always automatically wraps its result in a `GhostBox`

This is a more minimalistic design, the main drawbacks being that
- a ghost block cannot return unit (it returns `GhostBox<()>` instead), so sometimes a `;` must be appended at the end of the block
- we need an additional dereference in the case where the returned value is _already_ a `GhostBox`:
  ```rust
  let g: GhostBox<i32> = ...;
  let g2: GhostBox<i32> = ghost! {
      *g
  };
  ```
Overall I think these are acceptable, so I'll implement this change